js/data.js

```diff
@@ -936,6 +1112,17 @@ var data = {
         dozent: ["Dr. rer. nat. Harald Sack", "Jörg Waitelonis", "Nadine Steinmetz", "Magnus Knuth"],
         nameLV: "Semantic Multimedia"
     },
+    semanticMultimedia2: {
+        kurz: "Semantic Multimedia<br />(BPET)",
```

Comment:
I wondered too... but I didn't look up the lecture for the past summer term, where the Vertiefungsgebiete are supposedly different